walkbox.go

```diff
@@ -0,0 +1,105 @@
+package pctk
+
+import "log"
+
+const (
+	// MinPolygonVertices defines the minimum number of vertices to form a polygon.
+	MinPolygonVertices = 3
+)
+
+// Walkbox refers to a convex polygonal area that defines the walkable space for actors.
+type WalkBox struct {
+	WalkBoxID string
+	Enabled   bool
+	Vertices  []*Position
+	// TODO: Scale float32
```

Comment:
Please do not make any reference to this TODO. We don't know when scaling will be managed. Perhaps never ever.